packages/amesos2/src/Amesos2_Superludist_FunctionMap.hpp

```diff
@@ -372,10 +372,14 @@ namespace Amesos2 {
     static void LUstructInit(SLUD::int_t m, SLUD::int_t n,
 			     type_map::LUstruct_t* lu)
     {
+#if defined(AMESOS2_ENABLE_SUPERLUDIST_5_4_0)
```

Comment:
Is this set by users at configure time? Is it worth detecting the version at compile time via a test in `cmake/TPLs/FindTPLSuperLUDist.cmake` that would set an appropriate variable in `amesos2/cmake/Amesos2_config.h.in`? This isn't a request, just checking.

Reply:
This was a temporary flag and the flag should come with cmake_cxx_flags. I did in this way because we need to agree between application developers and test team about what version of superlu dist should be supported and tested. I am going to remove the flag as it won't be necessary anymore.